protocol/code_review.md

```diff
@@ -0,0 +1,75 @@
+Code Review
+===========
+
+A guide for reviewing code and having your code reviewed.
+
+Everyone
+--------
+
+* Accept that many programming decisions are opinions. Discuss tradeoffs, which
+  you prefer, and reach a resolution quickly.
+* Ask questions; don't make demands. ("What do you think about naming this
+  `:user_id`?")
+* Ask for clarification. ("I didn't understand. Can you clarify?")
+* Avoid selective ownership of code. ("mine", "not mine", "yours")
+* Avoid using terms that could be seen as referring to personal traits. ("dumb",
+  "stupid"). Assume everyone is attractive, intelligent, and well-meaning.
```

Comment:
The word "attractive" always jumps out at me here -- doesn't really seem relevant.